### ISSUE TITLE
A Length Mismatch in Recipes that use segmentation

### ISSUE THREAD

**Issue by South-Twilight** [kan-bayashi/ParallelWaveGAN#412]
Hi, thanks for the great repo.
When I run recipes that need to segment wav, just like OFUTON, CSD, ect., there's an error below:
```
Stage 1: Feature extraction
Feature extraction start. See the progress via dump/train/raw/preprocessing.*.log.
Feature extraction start. See the progress via dump/dev/raw/preprocessing.*.log.
Feature extraction start. See the progress via dump/eval/raw/preprocessing.*.log.
ERROR: wav.scp and segments has different #lines (47 vs 547).
ERROR: wav.scp and segments has different #lines (5 vs 70).
ERROR: wav.scp and segments has different #lines (5 vs 58).
```
It's a length mismatch between length before segment in `wav.scp` and length after segment in `segments`. I locate this error message at `utils/make_subset_data.sh` [L40-L54](https://github.com/kan-bayashi/ParallelWaveGAN/blob/master/utils/make_subset_data.sh#L40-L54).

How can I solve this problem? If I want to make `wav.scp` be the length after segmentation, what shuold I do?

**Comment by kan-bayashi** [comment]
Sorry for late reply. This is my mistake.
I will fix the issue.

**Comment by kan-bayashi** [comment]
Fixed in #413. Could you try again?

**Comment by South-Twilight** [comment]
> Fixed in #413. Could you try again?

It's ok to run now. Thanks.
